xgcm/grid.py

```diff
@@ -1080,11 +1080,12 @@ def __init__(
         face_connections : dict
             Grid topology
         coords : dict, optional
-            Explicit specification of axis coordinates, e.g
+            Explicit specification of axis dimensions, e.g
             ``{'X': {'center': 'XC', 'left: 'XG'}}``.
             Each key should be the name of an axis. The value should be
             a dictionary mapping positions (e.g. ``'left'``) to names of
-            coordinates in ``ds``.
+            axes in ``ds``. If the values are not present in ``ds`` or are not dimensions,
+            an error will be raised.
         metrics : dict, optional
             Specification of grid metrics
```

Comment:
`Specification of grid metrics mapping axis names (`X`, `Y`, `Z`) to corresponding metric variable names in the dataset`